### PR TITLE
Fix the metrics API call

### PR DESCRIPTION
**Pull request by bryanbellsmith**
### What's Changed?

- updated .gitignore for pytest
- added vscode settings for testing
- updated readme with testing section and minor typos
- bumped package version and added author
- moved all source code into the src folder
- fixed metrics API bug
- added pull request template

### PR Checklist
- [ ] Updated unit tests
- [X] Ran unit tests

### CODE REVIEW

**Review by a-guiducci — APPROVED** [review]
My admiration for the quality of this change is equaled only by my embarrassment that it was needed in the first place 🚀

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 __pycache__
+.pytest_cache
```
Comment (a-guiducci) [inline .gitignore:2]: no eof line 😤  literally unreviewable

**Review by a-guiducci — APPROVED** [review]
🚀

File: .gitignore (hunk 1 above)
Comment (bryanbellsmith) [reply .gitignore]: Fixed!

**Pull request merged**